docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -0,0 +1,210 @@
+==== {api-description-title}
+
+Use the get snapshot API to return information about one or more snapshots, including:
+
+* Value start and end time
+* Version of {es} that created the snapshot
+* List of included indices
+* Current state of the snapshot
+* List of failures that occurred during the snapshot
+
+[[get-snapshot-api-path-params]]
+==== {api-path-parms-title}
+
+`<repository>`::
+(Required, string)
+Comma-separated list of snapshot repository names used to limit the request.
+Wildcard (`*`) expressions are supported.
++
+To get information about all snapshot repositories registered in the
+cluster, omit this parameter or use `*` or `_all`.
+
+`<snapshot>`::
+(Required, string)
+Comma-separated list of snapshot names to retrieve. Also accepts wildcards (`*`).
++
+* To get information about all snapshots in a registered repository, use a wildcard (`*`) or `_all`.
+* To get information about any snapshots that are currently running, use `_current`.
++
+NOTE: Using `_all` in a request fails if any snapshots are unavailable.
+Set <<get-snapshot-api-ignore-unavailable,`ignore_unavailable`>> to `true` to return only available snapshots.
+
+[role="child-attributes"]
+[[get-snapshot-api-request-body]]
+==== {api-request-body-title}
+
+[[get-snapshot-api-ignore-unavailable]]
+`ignore_unavailable`::
+(Optional, boolean)
+If `false`, the request returns an error for any snapshots that are unavailable. Defaults to `false`.
++
+If `true`, the request ignores snapshots that are unavailable, such as those that are corrupted or temporarily cannot be returned.
+
+`verbose`::
+(Optional, boolean)
+If `true`, returns all available information about a snapshot. Defaults to `true`.
++
+If `false`, omits additional information about the snapshot, such as version information, start and end times, and the number of snapshotted shards.
+
+[role="child-attributes"]
+[[get-snapshot-api-response-body]]
+==== {api-response-body-title}
+
+`snapshot`::
+Name of the snapshot.
+
+`uuid`::
+Universally unique identifier (UUID) of the snapshot.
+
+`version_id`::
+Build ID of the {es} version used to create the snapshot.
+
+`version`::
+{es} version used to create the snapshot.
+
+`indices`::
+Indices that are included in the snapshot.
+
+`start_time`::
+Date timestamp of when the snapshot creation process started.
+
+`start_time_in_millis`::
+The start time, in milliseconds, when the snapshot creation process started.
+
+`end_time`::
+Date timestamp of when the snapshot creation process ended.
+
+`end_time_in_millis`::
+The end time, in milliseconds, when the snapshot creation process ended.
+
+`duration_in_millis`::
+How long, in milliseconds, it took to create the snapshot.
+
+`failures`::
+Lists any failures that occurred when creating the snapshot.
+
+`shards`::
+Indicates the following shards information:
+* Total shards included in the snapshot
+* Failed shards that were not included in the snapshot
+* Successful shards that were included in the snapshot
```

Comment:
You should be able to do a nested definition for this.
You can check out the search API's `shard` response param for an example:
https://www.elastic.co/guide/en/elasticsearch/reference/master/search-search.html#search-api-response-body

Reply:
Thanks! I'll apply this formatting.